hooks/command

```diff
@@ -360,6 +379,17 @@ fi
 
 command=()
 
+# Set up the copied workdir if copy-checkout
+if [[ "${BUILDKITE_PLUGIN_DOCKER_COPY_CHECKOUT:-off}" =~ ^(true|on|1)$ ]] ; then
+  COPY_COMMAND="cp -r ${workdir}_original/* ${workdir}"
```

Comment:
I don't think `${workdir}_original/*` will match hidden (dot) files/directories?

Reply:
I don't think `cp -r` preserves various attributes that might be important… you might want `cp -a`?

Also, could `$workdir` have spaces in it? In which case you might have trouble with it being interpreted as multiple separate arguments to `cp`. It sure gets tricky trying to glue together strings as commands in bash 😬